src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -749,3 +749,455 @@ def test_backup_archive (self, resource_group, vault_name, vm_name, storage_acco
             self.check("tierType", '{rp1_tier}'),
             self.check("resourceGroup", '{rg}')
         ])
+    # @ResourceGroupPreparer()
+    # @VaultPreparer()
+    # def test_backup_identity(self, resource_group, vault_name):
+    #     self.kwargs.update({
+    #         'vault': vault_name,
+    #         'rg': resource_group,
+    #     })
```

Comment:
If these tests are no longer needed, why not delete them directly?

Reply:
sure, will remove the commented tests